src/core/ext/filters/load_reporting/registered_opencensus_objects.h

```diff
@@ -30,28 +32,36 @@ namespace load_reporter {
 
 // Measures.
 
+// TODO(juanlishen): This seems weird, but without the dummy logging,
```

Comment:
I have a potential fix for this. But I need to understand the situation more before pushing it.

Reply:
Fixed.